这个bug算是暂时修复了吧

baimaoshouerkong opened the pull request:
这个问题有点棘手，目前能匹配到`"xx"` ,`xx "xx"`,`"xx" xx ` ,`"xx" "xx"`，最后一种`xx xx`的无法匹配完成，我的测试用例D:\Program Files\Locale.Emulator.2.5.0.1\LEP/roc.exe C:\Users\31193\Downloads\浏览器\ソラコイ\sorakoi.exe，最后匹配出D:\Program Files\Locale.Emulator.2.5.0.1\LEP/roc.exe C，但是exe C这个文件确实可以被创建（我试过了），我用cmd这个命令也报错。我也不知道怎么办。

Commit:
这个bug算是暂时修复了吧

这个问题有点棘手，目前能匹配到`"xx"` ,`xx "xx"`,`"xx" xx ` ,`"xx" "xx"`，最后一种`xx xx`的无法匹配完成，我的测试用例D:\Program Files\Locale.Emulator.2.5.0.1\LEP/roc.exe C:\Users\31193\Downloads\浏览器\ソラコイ\sorakoi.exe，最后匹配出D:\Program Files\Locale.Emulator.2.5.0.1\LEP/roc.exe C，但是exe C这个文件确实可以被创建（我试过了），我用cmd这个命令也报错。我也不知道怎么办。

GoldenPotato137 commented:
要不我们要求两个输入，把要执行的二进制与参数分开让玩家填入吧？

用一整个文本串拆分容易给自己识别制造不必要的困难（悲）

GoldenPotato137 commented:
太对啦，这就是目前的实现捏
![image](https://github.com/user-attachments/assets/d2e205ec-2c2b-4068-9f18-42aac296e5f0)

![image](https://github.com/user-attachments/assets/2d070d7b-a732-45ae-a64d-c586bf6f29f7)

baimaoshouerkong commented:
这样确实简单，不用吧正则表达式改了一遍又一遍

baimaoshouerkong commented:
那这个启动参数好像也不是那么需要OwO

GoldenPotato137 commented:
可能会有一些奇奇怪怪的使用场景，不直接调用可执行文件而要自己输入命令的，或许会用到自定义启动命令的功能。

这种情况要不就手写bat，要不我们就得提供一个类似于bat的功能。感觉明确区分要执行的文件和参数确实是比程序去猜靠谱点

Commit:
改好了

我试了一下没有发现什么问题，应该是可以了。这次加入了一个参数_startup_parameters_arguments这个参数代表这启动参数，这样子就和启动项分开了。其他应该就没有什么改动的地方了

Commit:
Merge branch 'dev' into fix_startup_parameters

baimaoshouerkong commented:
真是抱歉，这么简单的事情托得那么晚（主要是过年这几天事情太多了，人也不怎么在家），几分钟就搞定了，希望下次能有点挑战性

GoldenPotato137 commented:
目测没问题啦，感谢贡献❤️

p.s. 如果希望有挑战性可以来看看奇怪的COMException #328 问题 （bushi
